server/src/main/java/org/eclipse/openvsx/entities/ExtensionVersion.java

```diff
@@ -98,11 +93,13 @@
 
     String qna;
 
-    @ManyToMany
-    List<Extension> dependencies;
+    @Column
```

Comment:
not sure if we need more than 255 chars for the dependency list

Reply:
Yes, I think we should go with `1024`. 255 chars are exceeded relatively soon.

Reply:
Thx, I'm force-pushing it in a sec.